gnome/gnome-radio/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                gnome-radio
-version             0.2.0
+version             0.4.0
 revision            1
```

Comment:
When the version is increased, the revision should be set to 0.

Reply:
OK, added new pull request with the revision set to 0.